cmd/ddev/cmd/config.go

```diff
@@ -160,6 +160,8 @@ var ConfigCommand = &cobra.Command{
 			util.Failed("Could not write ddev config file: %v", err)
 		}
 
+		_,_ = app.CreateSettingsFile()
```

Comment:
We do have to capture and check the err here right? I don't think we need to capture the location of the file though for this context.